### ISSUE TITLE
unable to use `path` in list.jl (calling `gtk_tree_model_get_path`)

### ISSUE THREAD

**Issue by zweiglimmergneis** [JuliaGraphics/Gtk.jl#400]
Calling `path` in list.jl I get

> (julia103:6466): Gtk-CRITICAL **: 12:24:06.326: gtk_list_store_get_path: assertion 'iter->stamp == priv->stamp' failed
> (julia103:6466): Gtk-CRITICAL **: 12:24:06.326: gtk_tree_path_copy: assertion 'path != NULL' failed
ERROR: LoadError: objects of type getfield(Gtk, Symbol("##257#258")) cannot be finalized

Example:
``` julia
window = GtkWindow("issue path", 200, 100)
boxtop = GtkBox(:v) # vertical box, basic structure

wscroll = GtkScrolledWindow()

# create ls::GtkListStore and tv::GtkTreeView
ls, tv = itemlist([Int, String], ["No", "Name"])

push!(wscroll, tv)
push!(boxtop, wscroll)
push!(window, boxtop)

showall(window);

push!(ls, (3, "London"))
iter = push!(ls, (7, "Grenoble"))

pathend = Gtk.path(GtkTreeModel(ls), iter)
```
My overall goal is to make the last row in `ls` visible with a call to `gtk_tree_view_scroll_to_cell`, which works in C.

**Comment by tknopp** [comment]
thanks for the report. It seem the entire path handling is not tested. We thus need tests and fixes (to your problem.

I have found:
https://github.com/tknopp/Julietta.jl/blob/a90bb40042089157e490438f640404a3d256e6f1/src/workspace.jl#L42
so that at least something was working at a certain time.

A pull request would be very much appreciated.

**Comment by jonathanBieler** [comment]
I also got some issues with Trees today, there was some deprecation that weren't fixed. I still have some warning with GtkTreeIters and we should add some tests indeed. I can do it when I have time (next week probably).

**Comment by tknopp** [comment]
some tests would be very good!

**Comment by tknopp** [comment]
this was fixed by #401
